### ISSUE TITLE
Unable to update for 5.58 patch through quicklauncher

### ISSUE THREAD

**Issue by aRotondi1990** [goatcorp/FFXIVQuickLauncher#488]
As title - download is 'initiated' for patch 5.58, but after attempting to patch through quickLauncher, all downloads stall and do not progress. Swapping my runner to the default launcher (ffxivboot.exe) was able to apply the update and log in, and then switching back to quickLauncher allowed for regular playing / addons settings.

System Information:
OS: Arch Linux
Kernal v 5.13.3-181-tkg-bmq
Installed through Lutris using wine version lutris-5.7.11-x86_64

**Comment by reiichi001** [comment]
Hello!

In Lutris, please set the following environment variable:
`XL_WINEONLINUX=true`

After that, open the XIVLauncher settings, go tro the Patching tab, and make sure that the downloader is switched from `aria` to `.Net` as the new default patch downloader does not currently work on Linux. (The env var will ensure that wine-friendly default settings are used in the future, as we had handling for this, but the Lutris script wasn't providing it previously)

**Comment by WonderingDane** [comment]
> Hello!
> 
> In Lutris, please set the following environment variable:
> `XL_WINEONLINUX=true`
> 
> After that, open the XIVLauncher settings, go tro the Patching tab, and make sure that the downloader is switched from `aria` to `.Net` as the new default patch downloader does not currently work on Linux. (The env var will ensure that wine-friendly default settings are used in the future, as we had handling for this, but the Lutris script wasn't providing it previously)

What is mentioned above is what is default when installing with Lutris, but the download stops after a little while.
.NET(HTTP) is selected and the environment variable XL_WINEONLINUX=true is set in Lutris.

Weirdly, the download stops at different percentages every time.

System Information:
OS: POP!_OS 21.04
Kernel: 5.11.0-7620-generic
Installed through Lutris using wine version lutris-5.7.11-x86_64
![Screenshot from 2021-08-16 09-30-58](https://user-images.githubusercontent.com/18703455/129527793-075328b7-4de4-4530-8957-cf4dcfc43832.png)

**Comment by goaaats** [comment]
Yeah, that bug is why we replaced the downloader on Windows. I've been trying to figure it out, but haven't gotten anywhere, really should try to figure something out for Linux.

**Comment by reiichi001** [comment]
As of XIVLauncher 6.0.9 and with Wine 6.15 and later, you can now use the aria2c downloader option!

As the current iteration of the Lutris installer still using wine 6.10, we won't be changing the default Linux behavior at this time.
